cmd/manager/main.go

```diff
@@ -26,12 +27,18 @@ import (
 	machineactuator "sigs.k8s.io/cluster-api-provider-aws/pkg/actuators/machine"
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsproviderconfig/v1beta1"
 	awsclient "sigs.k8s.io/cluster-api-provider-aws/pkg/client"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/version"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
 func main() {
+	if len(os.Args) > 1 && os.Args[1] == "version" {
```

Comment:
We're already parsing the flags in order to initialize klog. It seems preferable to just use a flag for this:
```golang
var printVersion bool

func init() {
  flag.BoolVar(&printVersion, "version", false, "print version and exit")
}

func main() {
  ...
  flag.Parse()

  if printVersion {
    fmt.Println(version.String)
    os.Exit(0)
  }
}
```